--- old.anonymousClassDeclarationDoesntPrintWithReadonly.symbols
+++ new.anonymousClassDeclarationDoesntPrintWithReadonly.symbols
@@ -5,7 +5,7 @@
 >X : Symbol(X, Decl(anonymousClassDeclarationDoesntPrintWithReadonly.ts, 0, 0))
 
     constructor(readonly a: number) { }
->a : Symbol(X.a, Decl(anonymousClassDeclarationDoesntPrintWithReadonly.ts, 1, 16))
+>a : Symbol(a, Decl(anonymousClassDeclarationDoesntPrintWithReadonly.ts, 1, 16))
 }
 
 export function y() {
