--- old.arrayFakeFlatNoCrashInferenceDeclarations.symbols
+++ new.arrayFakeFlatNoCrashInferenceDeclarations.symbols
@@ -8,13 +8,13 @@
 >obj : Symbol(obj, Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 0, 48))
 
     "done": Arr,
->"done" : Symbol("done", Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 0, 54))
+>"done" : Symbol(done, Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 0, 54))
 >Arr : Symbol(Arr, Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 0, 18))
 
     "recur": Arr extends ReadonlyArray<infer InnerArr>
->"recur" : Symbol("recur", Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 1, 16))
+>"recur" : Symbol(recur, Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 1, 16))
 >Arr : Symbol(Arr, Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 0, 18))
->ReadonlyArray : Symbol(ReadonlyArray, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.core.d.ts, --, --), Decl(lib.es2015.iterable.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --), Decl(lib.es2016.array.include.d.ts, --, --) ... and 1 more)
+>ReadonlyArray : Symbol(ReadonlyArray)
 >InnerArr : Symbol(InnerArr, Decl(arrayFakeFlatNoCrashInferenceDeclarations.ts, 2, 44))
 
     ? BadFlatArray<InnerArr, [-1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20][Depth]>
