--- old.avoidListingPropertiesForTypesWithOnlyCallOrConstructSignatures.symbols
+++ new.avoidListingPropertiesForTypesWithOnlyCallOrConstructSignatures.symbols
@@ -5,7 +5,7 @@
 >Dog : Symbol(Dog, Decl(avoidListingPropertiesForTypesWithOnlyCallOrConstructSignatures.ts, 0, 0))
 
     barkable: true
->barkable : Symbol(Dog.barkable, Decl(avoidListingPropertiesForTypesWithOnlyCallOrConstructSignatures.ts, 0, 15))
+>barkable : Symbol(barkable, Decl(avoidListingPropertiesForTypesWithOnlyCallOrConstructSignatures.ts, 0, 15))
 }
 
 declare function getRover(): Dog
