--- old.capturedLetConstInLoop2.symbols
+++ new.capturedLetConstInLoop2.symbols
@@ -11,9 +11,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 3, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 2, 12))
@@ -34,9 +34,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 11, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 10, 12))
@@ -59,9 +59,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 19, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 18, 12))
@@ -80,9 +80,9 @@
     while (1 === 1) {
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 27, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 25, 14))
@@ -104,9 +104,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 36, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 35, 11))
@@ -130,9 +130,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 44, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         let x = 1;
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 45, 11))
@@ -159,9 +159,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 53, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 52, 12))
@@ -187,9 +187,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 63, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 62, 11))
@@ -214,9 +214,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 72, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 71, 11))
@@ -246,9 +246,9 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 82, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 81, 11))
@@ -271,9 +271,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 90, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 89, 14))
@@ -294,9 +294,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 98, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 97, 14))
@@ -318,9 +318,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 106, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 105, 14))
@@ -339,9 +339,9 @@
     while (1 === 1) {
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 114, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 112, 16))
@@ -363,9 +363,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 123, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 122, 13))
@@ -388,9 +388,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 131, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         const x = 1;
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 132, 13))
@@ -416,9 +416,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 140, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 139, 14))
@@ -444,9 +444,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 150, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 149, 13))
@@ -471,9 +471,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 159, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 158, 13))
@@ -502,9 +502,9 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2.ts, 169, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>arguments.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2.ts, 168, 13))
