--- old.capturedLetConstInLoop2_ES6.symbols
+++ new.capturedLetConstInLoop2_ES6.symbols
@@ -11,9 +11,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 3, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 2, 12))
@@ -34,9 +32,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 11, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 10, 12))
@@ -59,9 +55,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 19, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 18, 12))
@@ -80,9 +74,7 @@
     while (1 === 1) {
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 27, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 25, 14))
@@ -104,9 +96,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 36, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 35, 11))
@@ -130,9 +120,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 44, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         let x = 1;
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 45, 11))
@@ -159,9 +147,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 53, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 52, 12))
@@ -187,9 +173,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 63, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 62, 11))
@@ -214,9 +198,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 72, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 71, 11))
@@ -246,9 +228,7 @@
 
         let a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 82, 11))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 81, 11))
@@ -271,9 +251,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 90, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 89, 14))
@@ -294,9 +272,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 98, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 97, 14))
@@ -318,9 +294,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 106, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 105, 14))
@@ -339,9 +313,7 @@
     while (1 === 1) {
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 114, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 112, 16))
@@ -363,9 +335,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 123, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 122, 13))
@@ -388,9 +358,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 131, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         const x = 1;
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 132, 13))
@@ -416,9 +384,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 140, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 139, 14))
@@ -444,9 +410,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 150, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 149, 13))
@@ -471,9 +435,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 159, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 158, 13))
@@ -502,9 +464,7 @@
 
         const a = arguments.length;
 >a : Symbol(a, Decl(capturedLetConstInLoop2_ES6.ts, 169, 13))
->arguments.length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 >arguments : Symbol(arguments)
->length : Symbol(IArguments.length, Decl(lib.es5.d.ts, --, --))
 
         (function() { return x + y + a });
 >x : Symbol(x, Decl(capturedLetConstInLoop2_ES6.ts, 168, 13))
