--- old.caseInsensitiveFileSystemWithCapsImportTypeDeclarations.symbols
+++ new.caseInsensitiveFileSystemWithCapsImportTypeDeclarations.symbols
@@ -8,7 +8,7 @@
 >Broken : Symbol(Broken, Decl(index.ts, 0, 33))
 
     method () {
->method : Symbol(Broken.method, Decl(index.ts, 2, 21))
+>method : Symbol(method, Decl(index.ts, 2, 21))
 
         return { } as TypeB;
 >TypeB : Symbol(TypeB, Decl(index.ts, 0, 8))
