--- old.classExpressionInClassStaticDeclarations.symbols
+++ new.classExpressionInClassStaticDeclarations.symbols
@@ -5,6 +5,6 @@
 >C : Symbol(C, Decl(classExpressionInClassStaticDeclarations.ts, 0, 0))
 
     static D = class extends C {};
->D : Symbol(C.D, Decl(classExpressionInClassStaticDeclarations.ts, 0, 9))
+>D : Symbol(D, Decl(classExpressionInClassStaticDeclarations.ts, 0, 9))
 >C : Symbol(C, Decl(classExpressionInClassStaticDeclarations.ts, 0, 0))
 }
