--- old.cloduleWithPriorUninstantiatedModule.symbols
+++ new.cloduleWithPriorUninstantiatedModule.symbols
@@ -9,7 +9,7 @@
 >Someinterface : Symbol(Someinterface, Decl(cloduleWithPriorUninstantiatedModule.ts, 1, 18))
 
         foo(): void;
->foo : Symbol(Someinterface.foo, Decl(cloduleWithPriorUninstantiatedModule.ts, 2, 36))
+>foo : Symbol(foo, Decl(cloduleWithPriorUninstantiatedModule.ts, 2, 36))
     }
 }
 
