--- old.commentOnAmbientModule.symbols
+++ new.commentOnAmbientModule.symbols
@@ -7,12 +7,12 @@
 
     class foobar extends D.bar {
 >foobar : Symbol(foobar, Decl(b.ts, 1, 18))
->D.bar : Symbol(D.bar, Decl(a.ts, 11, 18))
+>D.bar : Symbol(bar, Decl(a.ts, 11, 18))
 >D : Symbol(D, Decl(a.ts, 8, 1))
->bar : Symbol(D.bar, Decl(a.ts, 11, 18))
+>bar : Symbol(bar, Decl(a.ts, 11, 18))
 
         foo();
->foo : Symbol(foobar.foo, Decl(b.ts, 2, 32))
+>foo : Symbol(foo, Decl(b.ts, 2, 32))
     }
 }
 === a.ts ===
