--- old.commentsOnReturnStatement1.symbols
+++ new.commentsOnReturnStatement1.symbols
@@ -5,7 +5,7 @@
 >DebugClass : Symbol(DebugClass, Decl(commentsOnReturnStatement1.ts, 0, 0))
 
     public static debugFunc() {
->debugFunc : Symbol(DebugClass.debugFunc, Decl(commentsOnReturnStatement1.ts, 0, 18))
+>debugFunc : Symbol(debugFunc, Decl(commentsOnReturnStatement1.ts, 0, 18))
 
         // Start Debugger Test Code
         var i = 0;
