--- old.conflictMarkerTrivia2.symbols
+++ new.conflictMarkerTrivia2.symbols
@@ -5,7 +5,7 @@
 >C : Symbol(C, Decl(conflictMarkerTrivia2.ts, 0, 0))
 
   foo() {
->foo : Symbol(C.foo, Decl(conflictMarkerTrivia2.ts, 0, 9))
+>foo : Symbol(foo, Decl(conflictMarkerTrivia2.ts, 0, 9))
 
 <<<<<<< B
      a();
@@ -16,6 +16,6 @@
 >>>>>>> A
 
   public bar() { }
->bar : Symbol(C.bar, Decl(conflictMarkerTrivia2.ts, 4, 3))
+>bar : Symbol(bar, Decl(conflictMarkerTrivia2.ts, 4, 3))
 }
 
