--- old.controlFlowWithIncompleteTypes.symbols
+++ new.controlFlowWithIncompleteTypes.symbols
@@ -20,9 +20,9 @@
 
             x = x.slice();
 >x : Symbol(x, Decl(controlFlowWithIncompleteTypes.ts, 5, 7))
->x.slice : Symbol(String.slice, Decl(lib.es5.d.ts, --, --))
+>x.slice : Symbol(slice, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(controlFlowWithIncompleteTypes.ts, 5, 7))
->slice : Symbol(String.slice, Decl(lib.es5.d.ts, --, --))
+>slice : Symbol(slice, Decl(lib.es5.d.ts, --, --))
         }
         else {
             x = "abc";
@@ -49,9 +49,9 @@
         else {
             x = x.slice();
 >x : Symbol(x, Decl(controlFlowWithIncompleteTypes.ts, 17, 7))
->x.slice : Symbol(String.slice, Decl(lib.es5.d.ts, --, --))
+>x.slice : Symbol(slice, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(controlFlowWithIncompleteTypes.ts, 17, 7))
->slice : Symbol(String.slice, Decl(lib.es5.d.ts, --, --))
+>slice : Symbol(slice, Decl(lib.es5.d.ts, --, --))
         }
     }
 }
