--- old.contextSensitiveReturnTypeInference.symbols
+++ new.contextSensitiveReturnTypeInference.symbols
@@ -7,7 +7,7 @@
 >IData : Symbol(IData, Decl(contextSensitiveReturnTypeInference.ts, 0, 0))
 
   bar: boolean
->bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 }
 
 declare function test<TDependencies>(
@@ -49,9 +49,9 @@
 
     fn2: data.bar
 >fn2 : Symbol(fn2, Decl(contextSensitiveReturnTypeInference.ts, 17, 40))
->data.bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>data.bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 >data : Symbol(data, Decl(contextSensitiveReturnTypeInference.ts, 16, 8))
->bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 
   }),
   DEPS
@@ -75,9 +75,9 @@
 
     fn2: data.bar
 >fn2 : Symbol(fn2, Decl(contextSensitiveReturnTypeInference.ts, 25, 40))
->data.bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>data.bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 >data : Symbol(data, Decl(contextSensitiveReturnTypeInference.ts, 24, 21))
->bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 
   }),
   DEPS
@@ -100,9 +100,9 @@
 
     fn2: data.bar
 >fn2 : Symbol(fn2, Decl(contextSensitiveReturnTypeInference.ts, 33, 24))
->data.bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>data.bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 >data : Symbol(data, Decl(contextSensitiveReturnTypeInference.ts, 32, 8))
->bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 
   }),
   DEPS
@@ -126,9 +126,9 @@
 
       fn2: data.bar
 >fn2 : Symbol(fn2, Decl(contextSensitiveReturnTypeInference.ts, 42, 32))
->data.bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>data.bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
 >data : Symbol(data, Decl(contextSensitiveReturnTypeInference.ts, 40, 8))
->bar : Symbol(IData.bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
+>bar : Symbol(bar, Decl(contextSensitiveReturnTypeInference.ts, 2, 17))
     }
   },
   DEPS
