--- old.contextuallyTypedJsxAttribute2.symbols
+++ new.contextuallyTypedJsxAttribute2.symbols
@@ -26,8 +26,6 @@
 
 ) {
   return <a></a>;
->a : Symbol(JSX.IntrinsicElements.a, Decl(react16.d.ts, 2516, 41))
->a : Symbol(JSX.IntrinsicElements.a, Decl(react16.d.ts, 2516, 41))
 }
 
 <UnwrappedLink onClick={(e) => {}} />;
@@ -56,8 +54,6 @@
   },
 ) {
   return <a></a>;
->a : Symbol(JSX.IntrinsicElements.a, Decl(react16.d.ts, 2516, 41))
->a : Symbol(JSX.IntrinsicElements.a, Decl(react16.d.ts, 2516, 41))
 }
 
 <UnwrappedLink2 onClick={(e) => {}} />;
