--- old.declarationEmitComputedNameCausesImportToBePainted.symbols
+++ new.declarationEmitComputedNameCausesImportToBePainted.symbols
@@ -3,13 +3,12 @@
 === context.ts ===
 export const Key = Symbol();
 >Key : Symbol(Key, Decl(context.ts, 0, 12))
->Symbol : Symbol(Symbol, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.symbol.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --))
 
 export interface Context {
 >Context : Symbol(Context, Decl(context.ts, 0, 28))
 
   [Key]: string;
->[Key] : Symbol(Context[Key], Decl(context.ts, 1, 26))
+>[Key] : Symbol([Key], Decl(context.ts, 1, 26))
 >Key : Symbol(Key, Decl(context.ts, 0, 12))
 }
 === index.ts ===
