--- old.declarationEmitForGlobalishSpecifierSymlink.symbols
+++ new.declarationEmitForGlobalishSpecifierSymlink.symbols
@@ -9,7 +9,7 @@
 >A : Symbol(A, Decl(impl.d.ts, 0, 26))
 
     Val
->Val : Symbol(A.Val, Decl(impl.d.ts, 1, 15))
+>Val : Symbol(Val, Decl(impl.d.ts, 1, 15))
 }
 === /p1/node_modules/typescript-fsa/index.d.ts ===
 
@@ -23,7 +23,7 @@
 >A : Symbol(A, Decl(impl.d.ts, 0, 26))
 
     Val
->Val : Symbol(A.Val, Decl(impl.d.ts, 1, 15))
+>Val : Symbol(Val, Decl(impl.d.ts, 1, 15))
 }
 === /p2/node_modules/typescript-fsa/index.d.ts ===
 
