--- old.declarationEmitPrivateNameCausesError.symbols
+++ new.declarationEmitPrivateNameCausesError.symbols
@@ -3,7 +3,6 @@
 === file.ts ===
 const IGNORE_EXTRA_VARIABLES = Symbol(); //Notice how this is unexported
 >IGNORE_EXTRA_VARIABLES : Symbol(IGNORE_EXTRA_VARIABLES, Decl(file.ts, 0, 5))
->Symbol : Symbol(Symbol, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.symbol.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --))
 
 //This is exported
 export function ignoreExtraVariables<CtorT extends {new(...args:any[]):{}}> (ctor : CtorT) {
@@ -17,7 +16,7 @@
 >ctor : Symbol(ctor, Decl(file.ts, 3, 77))
 
         [IGNORE_EXTRA_VARIABLES] = true; //An unexported constant is used
->[IGNORE_EXTRA_VARIABLES] : Symbol((Anonymous class)[IGNORE_EXTRA_VARIABLES], Decl(file.ts, 4, 31))
+>[IGNORE_EXTRA_VARIABLES] : Symbol([IGNORE_EXTRA_VARIABLES], Decl(file.ts, 4, 31))
 >IGNORE_EXTRA_VARIABLES : Symbol(IGNORE_EXTRA_VARIABLES, Decl(file.ts, 0, 5))
 
     };
