--- old.destructureComputedProperty.symbols
+++ new.destructureComputedProperty.symbols
@@ -16,7 +16,7 @@
 
 class C { private p: number; }
 >C : Symbol(C, Decl(destructureComputedProperty.ts, 2, 26))
->p : Symbol(C.p, Decl(destructureComputedProperty.ts, 4, 9))
+>p : Symbol(p, Decl(destructureComputedProperty.ts, 4, 9))
 
 const nameP = "p";
 >nameP : Symbol(nameP, Decl(destructureComputedProperty.ts, 5, 5))
@@ -36,7 +36,7 @@
 >C : Symbol(C, Decl(destructureComputedProperty.ts, 2, 26))
 
 const { p: p3 } = new C();
->p : Symbol(C.p, Decl(destructureComputedProperty.ts, 4, 9))
+>p : Symbol(p, Decl(destructureComputedProperty.ts, 4, 9))
 >p3 : Symbol(p3, Decl(destructureComputedProperty.ts, 9, 7))
 >C : Symbol(C, Decl(destructureComputedProperty.ts, 2, 26))
 
