--- old.detachedCommentAtStartOfLambdaFunction1.symbols
+++ new.detachedCommentAtStartOfLambdaFunction1.symbols
@@ -5,10 +5,10 @@
 >TestFile : Symbol(TestFile, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 0))
 
     name: string;
->name : Symbol(TestFile.name, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 16))
+>name : Symbol(name, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 16))
 
     foo(message: string): () => string {
->foo : Symbol(TestFile.foo, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 1, 17))
+>foo : Symbol(foo, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 1, 17))
 >message : Symbol(message, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 2, 8))
 
         return (...x: string[]) =>
@@ -19,8 +19,8 @@
             /// <returns type="Function" />
             message + this.name;
 >message : Symbol(message, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 2, 8))
->this.name : Symbol(TestFile.name, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 16))
+>this.name : Symbol(name, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 16))
 >this : Symbol(TestFile, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 0))
->name : Symbol(TestFile.name, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 16))
+>name : Symbol(name, Decl(detachedCommentAtStartOfLambdaFunction1.ts, 0, 16))
     }
 }
