--- old.discriminantPropertyInference.symbols
+++ new.discriminantPropertyInference.symbols
@@ -61,9 +61,9 @@
     cb: n => n.toFixed()
 >cb : Symbol(cb, Decl(discriminantPropertyInference.ts, 24, 16))
 >n : Symbol(n, Decl(discriminantPropertyInference.ts, 25, 7))
->n.toFixed : Symbol(Number.toFixed, Decl(lib.es5.d.ts, --, --))
+>n.toFixed : Symbol(toFixed, Decl(lib.es5.d.ts, --, --))
 >n : Symbol(n, Decl(discriminantPropertyInference.ts, 25, 7))
->toFixed : Symbol(Number.toFixed, Decl(lib.es5.d.ts, --, --))
+>toFixed : Symbol(toFixed, Decl(lib.es5.d.ts, --, --))
 
 });
 
@@ -78,9 +78,9 @@
     cb: n => n.toFixed()
 >cb : Symbol(cb, Decl(discriminantPropertyInference.ts, 30, 20))
 >n : Symbol(n, Decl(discriminantPropertyInference.ts, 31, 7))
->n.toFixed : Symbol(Number.toFixed, Decl(lib.es5.d.ts, --, --))
+>n.toFixed : Symbol(toFixed, Decl(lib.es5.d.ts, --, --))
 >n : Symbol(n, Decl(discriminantPropertyInference.ts, 31, 7))
->toFixed : Symbol(Number.toFixed, Decl(lib.es5.d.ts, --, --))
+>toFixed : Symbol(toFixed, Decl(lib.es5.d.ts, --, --))
 
 });
 
@@ -91,9 +91,9 @@
     cb: n => n.toFixed()
 >cb : Symbol(cb, Decl(discriminantPropertyInference.ts, 35, 3))
 >n : Symbol(n, Decl(discriminantPropertyInference.ts, 36, 7))
->n.toFixed : Symbol(Number.toFixed, Decl(lib.es5.d.ts, --, --))
+>n.toFixed : Symbol(toFixed, Decl(lib.es5.d.ts, --, --))
 >n : Symbol(n, Decl(discriminantPropertyInference.ts, 36, 7))
->toFixed : Symbol(Number.toFixed, Decl(lib.es5.d.ts, --, --))
+>toFixed : Symbol(toFixed, Decl(lib.es5.d.ts, --, --))
 
 });
 
