--- old.doNotElaborateAssignabilityToTypeParameters.symbols
+++ new.doNotElaborateAssignabilityToTypeParameters.symbols
@@ -6,7 +6,7 @@
 >T : Symbol(T, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 0, 19))
 >x : Symbol(x, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 0, 22))
 >T : Symbol(T, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 0, 19))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.iterable.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --))
+>Promise : Symbol(Promise)
 >T : Symbol(T, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 0, 19))
 
   let yaddable = await getXOrYadda(x);
@@ -22,10 +22,10 @@
 >Yadda : Symbol(Yadda, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 3, 1))
 
   stuff: string,
->stuff : Symbol(Yadda.stuff, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 5, 17))
+>stuff : Symbol(stuff, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 5, 17))
 
   things: string,
->things : Symbol(Yadda.things, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 6, 16))
+>things : Symbol(things, Decl(doNotElaborateAssignabilityToTypeParameters.ts, 6, 16))
 }
 
 declare function getXOrYadda<T>(x: T): T | Yadda;
