--- old.duplicateLocalVariable4.symbols
+++ new.duplicateLocalVariable4.symbols
@@ -5,7 +5,7 @@
 >E : Symbol(E, Decl(duplicateLocalVariable4.ts, 0, 0))
 
 a
->a : Symbol(E.a, Decl(duplicateLocalVariable4.ts, 0, 7))
+>a : Symbol(a, Decl(duplicateLocalVariable4.ts, 0, 7))
 }
  
 var x = E;
@@ -14,7 +14,7 @@
 
 var x = E.a;
 >x : Symbol(x, Decl(duplicateLocalVariable4.ts, 4, 3), Decl(duplicateLocalVariable4.ts, 5, 3))
->E.a : Symbol(E.a, Decl(duplicateLocalVariable4.ts, 0, 7))
+>E.a : Symbol(a, Decl(duplicateLocalVariable4.ts, 0, 7))
 >E : Symbol(E, Decl(duplicateLocalVariable4.ts, 0, 0))
->a : Symbol(E.a, Decl(duplicateLocalVariable4.ts, 0, 7))
+>a : Symbol(a, Decl(duplicateLocalVariable4.ts, 0, 7))
 
