--- old.es5-asyncFunctionReturnStatements.symbols
+++ new.es5-asyncFunctionReturnStatements.symbols
@@ -11,14 +11,14 @@
 
 async function returnStatement0(): Promise<any> {
 >returnStatement0 : Symbol(returnStatement0, Decl(es5-asyncFunctionReturnStatements.ts, 0, 29))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     return;
 }
 
 async function returnStatement1(): Promise<any> {
 >returnStatement1 : Symbol(returnStatement1, Decl(es5-asyncFunctionReturnStatements.ts, 4, 1))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     return x;
 >x : Symbol(x, Decl(es5-asyncFunctionReturnStatements.ts, 0, 11))
@@ -26,7 +26,7 @@
 
 async function returnStatement2(): Promise<any> {
 >returnStatement2 : Symbol(returnStatement2, Decl(es5-asyncFunctionReturnStatements.ts, 8, 1))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     return await x;
 >x : Symbol(x, Decl(es5-asyncFunctionReturnStatements.ts, 0, 11))
@@ -34,14 +34,14 @@
 
 async function returnStatement3(): Promise<any> {
 >returnStatement3 : Symbol(returnStatement3, Decl(es5-asyncFunctionReturnStatements.ts, 12, 1))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     { return; }
 }
 
 async function returnStatement4(): Promise<any> {
 >returnStatement4 : Symbol(returnStatement4, Decl(es5-asyncFunctionReturnStatements.ts, 16, 1))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     await x;
 >x : Symbol(x, Decl(es5-asyncFunctionReturnStatements.ts, 0, 11))
@@ -51,7 +51,7 @@
 
 async function returnStatement5(): Promise<any>{
 >returnStatement5 : Symbol(returnStatement5, Decl(es5-asyncFunctionReturnStatements.ts, 21, 1))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     { return await x; }
 >x : Symbol(x, Decl(es5-asyncFunctionReturnStatements.ts, 0, 11))
