--- old.es5-asyncFunctionTryStatements.symbols
+++ new.es5-asyncFunctionTryStatements.symbols
@@ -68,8 +68,8 @@
 
 async function tryCatch3(): Promise<Function> {
 >tryCatch3 : Symbol(tryCatch3, Decl(es5-asyncFunctionTryStatements.ts, 30, 1))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
->Function : Symbol(Function, Decl(lib.es5.d.ts, --, --), Decl(lib.es5.d.ts, --, --))
+>Promise : Symbol(Promise)
+>Function : Symbol(Function)
 
     var x: any, y: any;
 >x : Symbol(x, Decl(es5-asyncFunctionTryStatements.ts, 33, 7))
