--- old.esModuleInteropImportCall.symbols
+++ new.esModuleInteropImportCall.symbols
@@ -12,14 +12,9 @@
 
 === index.ts ===
 import("./foo").then(f => {
->import("./foo").then : Symbol(Promise.then, Decl(lib.es5.d.ts, --, --))
->"./foo" : Symbol("foo", Decl(foo.d.ts, 0, 0))
->then : Symbol(Promise.then, Decl(lib.es5.d.ts, --, --))
 >f : Symbol(f, Decl(index.ts, 0, 21))
 
     f.default;
->f.default : Symbol(default)
 >f : Symbol(f, Decl(index.ts, 0, 21))
->default : Symbol(default)
 
 });
