--- old.functionDeclarationWithArgumentOfTypeFunctionTypeArray.symbols
+++ new.functionDeclarationWithArgumentOfTypeFunctionTypeArray.symbols
@@ -7,8 +7,8 @@
 >x : Symbol(x, Decl(functionDeclarationWithArgumentOfTypeFunctionTypeArray.ts, 0, 22))
 
     return args.length;
->args.length : Symbol(Array.length, Decl(lib.es5.d.ts, --, --))
+>args.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >args : Symbol(args, Decl(functionDeclarationWithArgumentOfTypeFunctionTypeArray.ts, 0, 13))
->length : Symbol(Array.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 }
 
