--- old.genericTypeAssertions4.symbols
+++ new.genericTypeAssertions4.symbols
@@ -5,7 +5,7 @@
 >A : Symbol(A, Decl(genericTypeAssertions4.ts, 0, 0))
 
     foo() { return ""; }
->foo : Symbol(A.foo, Decl(genericTypeAssertions4.ts, 0, 9))
+>foo : Symbol(foo, Decl(genericTypeAssertions4.ts, 0, 9))
 }
 
 class B extends A {
@@ -13,7 +13,7 @@
 >A : Symbol(A, Decl(genericTypeAssertions4.ts, 0, 0))
 
     bar() { return 1; }
->bar : Symbol(B.bar, Decl(genericTypeAssertions4.ts, 4, 19))
+>bar : Symbol(bar, Decl(genericTypeAssertions4.ts, 4, 19))
 }
 
 class C extends A {
@@ -21,7 +21,7 @@
 >A : Symbol(A, Decl(genericTypeAssertions4.ts, 0, 0))
 
     baz() { return 1; }
->baz : Symbol(C.baz, Decl(genericTypeAssertions4.ts, 8, 19))
+>baz : Symbol(baz, Decl(genericTypeAssertions4.ts, 8, 19))
 }
 
 var a: A;
