--- old.genericWithCallSignatureReturningSpecialization.symbols
+++ new.genericWithCallSignatureReturningSpecialization.symbols
@@ -6,7 +6,7 @@
 >T : Symbol(T, Decl(genericWithCallSignatureReturningSpecialization.ts, 0, 12))
 
     f(): B<number>;
->f : Symbol(B.f, Decl(genericWithCallSignatureReturningSpecialization.ts, 0, 16))
+>f : Symbol(f, Decl(genericWithCallSignatureReturningSpecialization.ts, 0, 16))
 >B : Symbol(B, Decl(genericWithCallSignatureReturningSpecialization.ts, 0, 0))
 
     (value: T): void;
