--- old.impliedNodeFormatInterop1.symbols
+++ new.impliedNodeFormatInterop1.symbols
@@ -8,7 +8,7 @@
 >HighlightAPI : Symbol(HighlightAPI, Decl(index.d.ts, 0, 31))
 
     highlight(code: string): string;
->highlight : Symbol(HighlightAPI.highlight, Decl(index.d.ts, 1, 33))
+>highlight : Symbol(highlight, Decl(index.d.ts, 1, 33))
 >code : Symbol(code, Decl(index.d.ts, 2, 14))
   }
   const hljs: HighlightAPI;
@@ -31,7 +31,7 @@
 >hljs : Symbol(hljs, Decl(index.ts, 0, 6))
 
 hljs.highlight("code");
->hljs.highlight : Symbol(HighlightAPI.highlight, Decl(index.d.ts, 1, 33))
+>hljs.highlight : Symbol(highlight, Decl(index.d.ts, 1, 33))
 >hljs : Symbol(hljs, Decl(index.ts, 0, 6))
->highlight : Symbol(HighlightAPI.highlight, Decl(index.d.ts, 1, 33))
+>highlight : Symbol(highlight, Decl(index.d.ts, 1, 33))
 
