--- old.inheritanceGrandParentPrivateMemberCollision.symbols
+++ new.inheritanceGrandParentPrivateMemberCollision.symbols
@@ -5,7 +5,7 @@
 >A : Symbol(A, Decl(inheritanceGrandParentPrivateMemberCollision.ts, 0, 0))
 
     private myMethod() { }
->myMethod : Symbol(A.myMethod, Decl(inheritanceGrandParentPrivateMemberCollision.ts, 0, 9))
+>myMethod : Symbol(myMethod, Decl(inheritanceGrandParentPrivateMemberCollision.ts, 0, 9))
 }
 
 class B extends A { }
@@ -17,6 +17,6 @@
 >B : Symbol(B, Decl(inheritanceGrandParentPrivateMemberCollision.ts, 2, 1))
 
     private myMethod() { }
->myMethod : Symbol(C.myMethod, Decl(inheritanceGrandParentPrivateMemberCollision.ts, 6, 19))
+>myMethod : Symbol(myMethod, Decl(inheritanceGrandParentPrivateMemberCollision.ts, 6, 19))
 }
 
