--- old.inheritanceMemberFuncOverridingAccessor.symbols
+++ new.inheritanceMemberFuncOverridingAccessor.symbols
@@ -5,12 +5,12 @@
 >a : Symbol(a, Decl(inheritanceMemberFuncOverridingAccessor.ts, 0, 0))
 
     get x() {
->x : Symbol(a.x, Decl(inheritanceMemberFuncOverridingAccessor.ts, 0, 9), Decl(inheritanceMemberFuncOverridingAccessor.ts, 3, 5))
+>x : Symbol(x, Decl(inheritanceMemberFuncOverridingAccessor.ts, 0, 9), Decl(inheritanceMemberFuncOverridingAccessor.ts, 3, 5))
 
         return "20";
     }
     set x(aValue: string) {
->x : Symbol(a.x, Decl(inheritanceMemberFuncOverridingAccessor.ts, 0, 9), Decl(inheritanceMemberFuncOverridingAccessor.ts, 3, 5))
+>x : Symbol(x, Decl(inheritanceMemberFuncOverridingAccessor.ts, 0, 9), Decl(inheritanceMemberFuncOverridingAccessor.ts, 3, 5))
 >aValue : Symbol(aValue, Decl(inheritanceMemberFuncOverridingAccessor.ts, 4, 10))
 
     }
@@ -21,7 +21,7 @@
 >a : Symbol(a, Decl(inheritanceMemberFuncOverridingAccessor.ts, 0, 0))
 
     x() {
->x : Symbol(b.x, Decl(inheritanceMemberFuncOverridingAccessor.ts, 9, 19))
+>x : Symbol(x, Decl(inheritanceMemberFuncOverridingAccessor.ts, 9, 19))
 
         return "20";
     }
