--- old.inheritanceStaticPropertyOverridingProperty.symbols
+++ new.inheritanceStaticPropertyOverridingProperty.symbols
@@ -5,7 +5,7 @@
 >a : Symbol(a, Decl(inheritanceStaticPropertyOverridingProperty.ts, 0, 0))
 
     static x: () => string;
->x : Symbol(a.x, Decl(inheritanceStaticPropertyOverridingProperty.ts, 0, 9))
+>x : Symbol(x, Decl(inheritanceStaticPropertyOverridingProperty.ts, 0, 9))
 }
 
 class b extends a {
@@ -13,5 +13,5 @@
 >a : Symbol(a, Decl(inheritanceStaticPropertyOverridingProperty.ts, 0, 0))
 
     static x: () => string;
->x : Symbol(b.x, Decl(inheritanceStaticPropertyOverridingProperty.ts, 4, 19))
+>x : Symbol(x, Decl(inheritanceStaticPropertyOverridingProperty.ts, 4, 19))
 }
