--- old.inheritanceMemberPropertyOverridingProperty.symbols
+++ new.inheritanceMemberPropertyOverridingProperty.symbols
@@ -5,7 +5,7 @@
 >a : Symbol(a, Decl(inheritanceMemberPropertyOverridingProperty.ts, 0, 0))
 
     x: () => string;
->x : Symbol(a.x, Decl(inheritanceMemberPropertyOverridingProperty.ts, 0, 9))
+>x : Symbol(x, Decl(inheritanceMemberPropertyOverridingProperty.ts, 0, 9))
 }
 
 class b extends a {
@@ -13,5 +13,5 @@
 >a : Symbol(a, Decl(inheritanceMemberPropertyOverridingProperty.ts, 0, 0))
 
     x: () => string;
->x : Symbol(b.x, Decl(inheritanceMemberPropertyOverridingProperty.ts, 4, 19))
+>x : Symbol(x, Decl(inheritanceMemberPropertyOverridingProperty.ts, 4, 19))
 }
