--- old.instanceofOnInstantiationExpression.symbols
+++ new.instanceofOnInstantiationExpression.symbols
@@ -6,7 +6,7 @@
 >T : Symbol(T, Decl(instanceofOnInstantiationExpression.ts, 0, 18))
 
     value: T;
->value : Symbol(Box.value, Decl(instanceofOnInstantiationExpression.ts, 0, 22))
+>value : Symbol(value, Decl(instanceofOnInstantiationExpression.ts, 0, 22))
 >T : Symbol(T, Decl(instanceofOnInstantiationExpression.ts, 0, 18))
 }
 
