--- old.isolatedDeclarationErrorsClassesExpressions.symbols
+++ new.isolatedDeclarationErrorsClassesExpressions.symbols
@@ -5,7 +5,7 @@
 >cls : Symbol(cls, Decl(isolatedDeclarationErrorsClassesExpressions.ts, 0, 12))
 
     foo: string = "";
->foo : Symbol(cls.foo, Decl(isolatedDeclarationErrorsClassesExpressions.ts, 0, 26))
+>foo : Symbol(foo, Decl(isolatedDeclarationErrorsClassesExpressions.ts, 0, 26))
 }
 
 
