--- old.jsFileClassSelfReferencedProperty.symbols
+++ new.jsFileClassSelfReferencedProperty.symbols
@@ -6,12 +6,8 @@
 
   constructor () {
     this.testStackOverflow = this.testStackOverflow.bind(this)
->this.testStackOverflow : Symbol(StackOverflowTest.testStackOverflow, Decl(foo.js, 1, 18))
 >this : Symbol(StackOverflowTest, Decl(foo.js, 0, 0))
->testStackOverflow : Symbol(StackOverflowTest.testStackOverflow, Decl(foo.js, 1, 18))
->this.testStackOverflow : Symbol(StackOverflowTest.testStackOverflow, Decl(foo.js, 1, 18))
 >this : Symbol(StackOverflowTest, Decl(foo.js, 0, 0))
->testStackOverflow : Symbol(StackOverflowTest.testStackOverflow, Decl(foo.js, 1, 18))
 >this : Symbol(StackOverflowTest, Decl(foo.js, 0, 0))
   }
 }
