--- old.keyofDoesntContainSymbols.symbols
+++ new.keyofDoesntContainSymbols.symbols
@@ -3,7 +3,6 @@
 === keyofDoesntContainSymbols.ts ===
 const sym = Symbol();
 >sym : Symbol(sym, Decl(keyofDoesntContainSymbols.ts, 0, 5))
->Symbol : Symbol(Symbol, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.symbol.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --))
 
 const num = 0;
 >num : Symbol(num, Decl(keyofDoesntContainSymbols.ts, 1, 5))
