--- old.misspelledJsDocTypedefTags.symbols
+++ new.misspelledJsDocTypedefTags.symbols
@@ -3,9 +3,9 @@
 === a.js ===
 /** @typedef {{ endTime: number, screenshots: number}} A.<b>*/
 Animation.AnimationModel.ScreenshotCapture.Request;
->Animation : Symbol(Animation, Decl(lib.dom.d.ts, --, --), Decl(lib.dom.d.ts, --, --), Decl(a.js, 1, 51))
+>Animation : Symbol(Animation, Decl(lib.dom.d.ts, --, --), Decl(lib.dom.d.ts, --, --))
 
 /** @typedef {{ endTime: number, screenshots: !B.<string>}} */
 Animation.AnimationModel.ScreenshotCapture.Request;
->Animation : Symbol(Animation, Decl(lib.dom.d.ts, --, --), Decl(lib.dom.d.ts, --, --), Decl(a.js, 1, 51))
+>Animation : Symbol(Animation, Decl(lib.dom.d.ts, --, --), Decl(lib.dom.d.ts, --, --))
 
