--- old.moduleMerge.symbols
+++ new.moduleMerge.symbols
@@ -10,7 +10,7 @@
 >B : Symbol(B, Decl(moduleMerge.ts, 3, 1))
     {
         public Hello(): string
->Hello : Symbol(B.Hello, Decl(moduleMerge.ts, 5, 5))
+>Hello : Symbol(Hello, Decl(moduleMerge.ts, 5, 5))
         {
             return "from private B";
         }
@@ -24,7 +24,7 @@
 >B : Symbol(B, Decl(moduleMerge.ts, 14, 1))
     {
         public Hello(): string
->Hello : Symbol(B.Hello, Decl(moduleMerge.ts, 16, 5))
+>Hello : Symbol(Hello, Decl(moduleMerge.ts, 16, 5))
         {
             return "from export B";
         }
