--- old.moduleResolutionWithSymlinks_withOutDir.symbols
+++ new.moduleResolutionWithSymlinks_withOutDir.symbols
@@ -26,7 +26,7 @@
 === /src/library-a/index.ts ===
 export class MyClass { private x: number; }
 >MyClass : Symbol(MyClass, Decl(index.ts, 0, 0))
->x : Symbol(MyClass.x, Decl(index.ts, 0, 22))
+>x : Symbol(x, Decl(index.ts, 0, 22))
 
 === /src/library-b/index.ts ===
 import {MyClass} from "library-a";
