--- old.moduleMemberWithoutTypeAnnotation2.symbols
+++ new.moduleMemberWithoutTypeAnnotation2.symbols
@@ -11,7 +11,7 @@
 >IDiagnosticWriter : Symbol(IDiagnosticWriter, Decl(moduleMemberWithoutTypeAnnotation2.ts, 1, 39))
 
             Alert(output: string): void;
->Alert : Symbol(IDiagnosticWriter.Alert, Decl(moduleMemberWithoutTypeAnnotation2.ts, 3, 44))
+>Alert : Symbol(Alert, Decl(moduleMemberWithoutTypeAnnotation2.ts, 3, 44))
 >output : Symbol(output, Decl(moduleMemberWithoutTypeAnnotation2.ts, 4, 18))
         }
 
