--- old.narrowingOfDottedNames.symbols
+++ new.narrowingOfDottedNames.symbols
@@ -7,7 +7,7 @@
 >A : Symbol(A, Decl(narrowingOfDottedNames.ts, 0, 0))
 
     prop!: { a: string; };
->prop : Symbol(A.prop, Decl(narrowingOfDottedNames.ts, 2, 9))
+>prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 2, 9))
 >a : Symbol(a, Decl(narrowingOfDottedNames.ts, 3, 12))
 }
 
@@ -15,7 +15,7 @@
 >B : Symbol(B, Decl(narrowingOfDottedNames.ts, 4, 1))
 
     prop!: { b: string; }
->prop : Symbol(B.prop, Decl(narrowingOfDottedNames.ts, 6, 9))
+>prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 6, 9))
 >b : Symbol(b, Decl(narrowingOfDottedNames.ts, 7, 12))
 }
 
@@ -54,9 +54,9 @@
 
             x.prop.a;
 >x.prop.a : Symbol(a, Decl(narrowingOfDottedNames.ts, 3, 12))
->x.prop : Symbol(A.prop, Decl(narrowingOfDottedNames.ts, 2, 9))
+>x.prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 2, 9))
 >x : Symbol(x, Decl(narrowingOfDottedNames.ts, 18, 12))
->prop : Symbol(A.prop, Decl(narrowingOfDottedNames.ts, 2, 9))
+>prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 2, 9))
 >a : Symbol(a, Decl(narrowingOfDottedNames.ts, 3, 12))
         }
         else if (x instanceof B) {
@@ -65,9 +65,9 @@
 
             x.prop.b;
 >x.prop.b : Symbol(b, Decl(narrowingOfDottedNames.ts, 7, 12))
->x.prop : Symbol(B.prop, Decl(narrowingOfDottedNames.ts, 6, 9))
+>x.prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 6, 9))
 >x : Symbol(x, Decl(narrowingOfDottedNames.ts, 18, 12))
->prop : Symbol(B.prop, Decl(narrowingOfDottedNames.ts, 6, 9))
+>prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 6, 9))
 >b : Symbol(b, Decl(narrowingOfDottedNames.ts, 7, 12))
         }
     }
@@ -86,9 +86,9 @@
 
             x.prop.a;
 >x.prop.a : Symbol(a, Decl(narrowingOfDottedNames.ts, 3, 12))
->x.prop : Symbol(A.prop, Decl(narrowingOfDottedNames.ts, 2, 9))
+>x.prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 2, 9))
 >x : Symbol(x, Decl(narrowingOfDottedNames.ts, 29, 12))
->prop : Symbol(A.prop, Decl(narrowingOfDottedNames.ts, 2, 9))
+>prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 2, 9))
 >a : Symbol(a, Decl(narrowingOfDottedNames.ts, 3, 12))
         }
         else if (isB(x)) {
@@ -97,9 +97,9 @@
 
             x.prop.b;
 >x.prop.b : Symbol(b, Decl(narrowingOfDottedNames.ts, 7, 12))
->x.prop : Symbol(B.prop, Decl(narrowingOfDottedNames.ts, 6, 9))
+>x.prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 6, 9))
 >x : Symbol(x, Decl(narrowingOfDottedNames.ts, 29, 12))
->prop : Symbol(B.prop, Decl(narrowingOfDottedNames.ts, 6, 9))
+>prop : Symbol(prop, Decl(narrowingOfDottedNames.ts, 6, 9))
 >b : Symbol(b, Decl(narrowingOfDottedNames.ts, 7, 12))
         }
     }
@@ -111,7 +111,7 @@
 >Foo1 : Symbol(Foo1, Decl(narrowingOfDottedNames.ts, 38, 1))
 {
     x: number;  // Error
->x : Symbol(Foo1.x, Decl(narrowingOfDottedNames.ts, 43, 1))
+>x : Symbol(x, Decl(narrowingOfDottedNames.ts, 43, 1))
 
     constructor() {
         if (this instanceof Boolean) {
@@ -125,7 +125,7 @@
 >Foo2 : Symbol(Foo2, Decl(narrowingOfDottedNames.ts, 49, 1))
 {
     x: number;  // Error
->x : Symbol(Foo2.x, Decl(narrowingOfDottedNames.ts, 52, 1))
+>x : Symbol(x, Decl(narrowingOfDottedNames.ts, 52, 1))
 
     constructor() {
     }
@@ -137,21 +137,21 @@
 >AInfo : Symbol(AInfo, Decl(narrowingOfDottedNames.ts, 56, 1))
 
     a_count: number = 1;
->a_count : Symbol(AInfo.a_count, Decl(narrowingOfDottedNames.ts, 60, 13))
+>a_count : Symbol(a_count, Decl(narrowingOfDottedNames.ts, 60, 13))
 }
 
 class BInfo {
 >BInfo : Symbol(BInfo, Decl(narrowingOfDottedNames.ts, 62, 1))
 
     b_count: number = 1;
->b_count : Symbol(BInfo.b_count, Decl(narrowingOfDottedNames.ts, 64, 13))
+>b_count : Symbol(b_count, Decl(narrowingOfDottedNames.ts, 64, 13))
 }
 
 class Base {
 >Base : Symbol(Base, Decl(narrowingOfDottedNames.ts, 66, 1))
 
     id: number = 0;
->id : Symbol(Base.id, Decl(narrowingOfDottedNames.ts, 68, 12))
+>id : Symbol(id, Decl(narrowingOfDottedNames.ts, 68, 12))
 }
 
 class A2 extends Base {
@@ -159,7 +159,7 @@
 >Base : Symbol(Base, Decl(narrowingOfDottedNames.ts, 66, 1))
 
     info!: AInfo;
->info : Symbol(A2.info, Decl(narrowingOfDottedNames.ts, 72, 23))
+>info : Symbol(info, Decl(narrowingOfDottedNames.ts, 72, 23))
 >AInfo : Symbol(AInfo, Decl(narrowingOfDottedNames.ts, 56, 1))
 }
 
@@ -168,7 +168,7 @@
 >Base : Symbol(Base, Decl(narrowingOfDottedNames.ts, 66, 1))
 
     info!: BInfo;
->info : Symbol(B2.info, Decl(narrowingOfDottedNames.ts, 76, 23))
+>info : Symbol(info, Decl(narrowingOfDottedNames.ts, 76, 23))
 >BInfo : Symbol(BInfo, Decl(narrowingOfDottedNames.ts, 62, 1))
 }
 
@@ -184,11 +184,11 @@
 >A2 : Symbol(A2, Decl(narrowingOfDottedNames.ts, 70, 1))
 
         target.info.a_count = 3;
->target.info.a_count : Symbol(AInfo.a_count, Decl(narrowingOfDottedNames.ts, 60, 13))
->target.info : Symbol(A2.info, Decl(narrowingOfDottedNames.ts, 72, 23))
+>target.info.a_count : Symbol(a_count, Decl(narrowingOfDottedNames.ts, 60, 13))
+>target.info : Symbol(info, Decl(narrowingOfDottedNames.ts, 72, 23))
 >target : Symbol(target, Decl(narrowingOfDottedNames.ts, 80, 3))
->info : Symbol(A2.info, Decl(narrowingOfDottedNames.ts, 72, 23))
->a_count : Symbol(AInfo.a_count, Decl(narrowingOfDottedNames.ts, 60, 13))
+>info : Symbol(info, Decl(narrowingOfDottedNames.ts, 72, 23))
+>a_count : Symbol(a_count, Decl(narrowingOfDottedNames.ts, 60, 13))
     }
     else if (target instanceof B2) {
 >target : Symbol(target, Decl(narrowingOfDottedNames.ts, 80, 3))
@@ -197,9 +197,9 @@
         const j: BInfo = target.info;
 >j : Symbol(j, Decl(narrowingOfDottedNames.ts, 87, 13))
 >BInfo : Symbol(BInfo, Decl(narrowingOfDottedNames.ts, 62, 1))
->target.info : Symbol(B2.info, Decl(narrowingOfDottedNames.ts, 76, 23))
+>target.info : Symbol(info, Decl(narrowingOfDottedNames.ts, 76, 23))
 >target : Symbol(target, Decl(narrowingOfDottedNames.ts, 80, 3))
->info : Symbol(B2.info, Decl(narrowingOfDottedNames.ts, 76, 23))
+>info : Symbol(info, Decl(narrowingOfDottedNames.ts, 76, 23))
     }
 }
 
