--- old.nestedRecursiveArraysOrObjectsError01.symbols
+++ new.nestedRecursiveArraysOrObjectsError01.symbols
@@ -15,7 +15,7 @@
 >StyleBase : Symbol(StyleBase, Decl(nestedRecursiveArraysOrObjectsError01.ts, 1, 44))
 
     foo: string;
->foo : Symbol(StyleBase.foo, Decl(nestedRecursiveArraysOrObjectsError01.ts, 2, 21))
+>foo : Symbol(foo, Decl(nestedRecursiveArraysOrObjectsError01.ts, 2, 21))
 }
 
 const blah: Style = [
