--- old.noImplicitAnyInBareInterface.symbols
+++ new.noImplicitAnyInBareInterface.symbols
@@ -7,8 +7,8 @@
     // Should return error for implicit any on `new` and `foo`.
     new ();
     few() : any;
->few : Symbol(Entry.few, Decl(noImplicitAnyInBareInterface.ts, 2, 11))
+>few : Symbol(few, Decl(noImplicitAnyInBareInterface.ts, 2, 11))
 
     foo();
->foo : Symbol(Entry.foo, Decl(noImplicitAnyInBareInterface.ts, 3, 16))
+>foo : Symbol(foo, Decl(noImplicitAnyInBareInterface.ts, 3, 16))
 }
