--- old.outModuleTripleSlashRefs.symbols
+++ new.outModuleTripleSlashRefs.symbols
@@ -6,7 +6,7 @@
 >A : Symbol(A, Decl(a.ts, 0, 0))
 
 	member: typeof GlobalFoo;
->member : Symbol(A.member, Decl(a.ts, 1, 16))
+>member : Symbol(member, Decl(a.ts, 1, 16))
 >GlobalFoo : Symbol(GlobalFoo, Decl(b.ts, 4, 11))
 }
 
@@ -16,7 +16,7 @@
 >Foo : Symbol(Foo, Decl(b.ts, 0, 0))
 
 	member: Bar;
->member : Symbol(Foo.member, Decl(b.ts, 1, 11))
+>member : Symbol(member, Decl(b.ts, 1, 11))
 >Bar : Symbol(Bar, Decl(c.d.ts, 0, 0))
 }
 declare var GlobalFoo: Foo;
@@ -29,7 +29,7 @@
 >Bar : Symbol(Bar, Decl(c.d.ts, 0, 0))
 
 	member: Baz;
->member : Symbol(Bar.member, Decl(c.d.ts, 1, 19))
+>member : Symbol(member, Decl(c.d.ts, 1, 19))
 >Baz : Symbol(Baz, Decl(d.d.ts, 0, 0))
 }
 
@@ -38,7 +38,7 @@
 >Baz : Symbol(Baz, Decl(d.d.ts, 0, 0))
 
 	member: number;
->member : Symbol(Baz.member, Decl(d.d.ts, 0, 19))
+>member : Symbol(member, Decl(d.d.ts, 0, 19))
 }
 
 === b.ts ===
