--- old.parameterReferencesOtherParameter1.symbols
+++ new.parameterReferencesOtherParameter1.symbols
@@ -5,7 +5,7 @@
 >Model : Symbol(Model, Decl(parameterReferencesOtherParameter1.ts, 0, 0))
 
     public name: string;
->name : Symbol(Model.name, Decl(parameterReferencesOtherParameter1.ts, 0, 13))
+>name : Symbol(name, Decl(parameterReferencesOtherParameter1.ts, 0, 13))
 }
 
 class UI {
@@ -15,9 +15,9 @@
 >model : Symbol(model, Decl(parameterReferencesOtherParameter1.ts, 5, 16))
 >Model : Symbol(Model, Decl(parameterReferencesOtherParameter1.ts, 0, 0))
 >foo : Symbol(foo, Decl(parameterReferencesOtherParameter1.ts, 5, 29))
->model.name : Symbol(Model.name, Decl(parameterReferencesOtherParameter1.ts, 0, 13))
+>model.name : Symbol(name, Decl(parameterReferencesOtherParameter1.ts, 0, 13))
 >model : Symbol(model, Decl(parameterReferencesOtherParameter1.ts, 5, 16))
->name : Symbol(Model.name, Decl(parameterReferencesOtherParameter1.ts, 0, 13))
+>name : Symbol(name, Decl(parameterReferencesOtherParameter1.ts, 0, 13))
     {
     }
 }
