--- old.privacyClass.symbols
+++ new.privacyClass.symbols
@@ -16,7 +16,7 @@
 >m1_c_public : Symbol(m1_c_public, Decl(privacyClass.ts, 5, 5))
 
         private f1() {
->f1 : Symbol(m1_c_public.f1, Decl(privacyClass.ts, 7, 30))
+>f1 : Symbol(f1, Decl(privacyClass.ts, 7, 30))
         }
     }
 
@@ -100,7 +100,7 @@
 >m2_c_public : Symbol(m2_c_public, Decl(privacyClass.ts, 49, 5))
 
         private f1() {
->f1 : Symbol(m2_c_public.f1, Decl(privacyClass.ts, 51, 30))
+>f1 : Symbol(f1, Decl(privacyClass.ts, 51, 30))
         }
     }
 
@@ -180,7 +180,7 @@
 >glo_c_public : Symbol(glo_c_public, Decl(privacyClass.ts, 91, 1))
 
     private f1() {
->f1 : Symbol(glo_c_public.f1, Decl(privacyClass.ts, 93, 27))
+>f1 : Symbol(f1, Decl(privacyClass.ts, 93, 27))
     }
 }
 
