--- old.recursiveTypes1.symbols
+++ new.recursiveTypes1.symbols
@@ -8,11 +8,11 @@
 >T : Symbol(T, Decl(recursiveTypes1.ts, 0, 17))
 
    X: T;
->X : Symbol(Entity.X, Decl(recursiveTypes1.ts, 0, 39))
+>X : Symbol(X, Decl(recursiveTypes1.ts, 0, 39))
 >T : Symbol(T, Decl(recursiveTypes1.ts, 0, 17))
 
    Y: T;
->Y : Symbol(Entity.Y, Decl(recursiveTypes1.ts, 1, 8))
+>Y : Symbol(Y, Decl(recursiveTypes1.ts, 1, 8))
 >T : Symbol(T, Decl(recursiveTypes1.ts, 0, 17))
 }
 
@@ -25,7 +25,7 @@
 >U : Symbol(U, Decl(recursiveTypes1.ts, 5, 17))
 
 	n: number;
->n : Symbol(Person.n, Decl(recursiveTypes1.ts, 5, 57))
+>n : Symbol(n, Decl(recursiveTypes1.ts, 5, 57))
 }
 
 interface Customer extends Person<Customer> {
@@ -34,6 +34,6 @@
 >Customer : Symbol(Customer, Decl(recursiveTypes1.ts, 7, 1))
 
 	s: string;
->s : Symbol(Customer.s, Decl(recursiveTypes1.ts, 9, 45))
+>s : Symbol(s, Decl(recursiveTypes1.ts, 9, 45))
 }
 
