--- old.recursivelySpecializedConstructorDeclaration.symbols
+++ new.recursivelySpecializedConstructorDeclaration.symbols
@@ -13,7 +13,7 @@
 
         // Removing this line fixes the constructor of ItemValue
         options: ViewModel<TValue>;
->options : Symbol(Interface.options, Decl(recursivelySpecializedConstructorDeclaration.ts, 2, 40))
+>options : Symbol(options, Decl(recursivelySpecializedConstructorDeclaration.ts, 2, 40))
 >ViewModel : Symbol(ViewModel, Decl(recursivelySpecializedConstructorDeclaration.ts, 10, 5))
 >TValue : Symbol(TValue, Decl(recursivelySpecializedConstructorDeclaration.ts, 2, 31))
     }    
