--- old.staticMemberAccessOffDerivedType1.symbols
+++ new.staticMemberAccessOffDerivedType1.symbols
@@ -5,7 +5,7 @@
 >SomeBase : Symbol(SomeBase, Decl(staticMemberAccessOffDerivedType1.ts, 0, 0))
 
     static GetNumber() {
->GetNumber : Symbol(SomeBase.GetNumber, Decl(staticMemberAccessOffDerivedType1.ts, 0, 16))
+>GetNumber : Symbol(GetNumber, Decl(staticMemberAccessOffDerivedType1.ts, 0, 16))
 
         return 2;
     }
@@ -15,9 +15,9 @@
 >SomeBase : Symbol(SomeBase, Decl(staticMemberAccessOffDerivedType1.ts, 0, 0))
 
     static SomeNumber = P.GetNumber();
->SomeNumber : Symbol(P.SomeNumber, Decl(staticMemberAccessOffDerivedType1.ts, 5, 26))
->P.GetNumber : Symbol(SomeBase.GetNumber, Decl(staticMemberAccessOffDerivedType1.ts, 0, 16))
+>SomeNumber : Symbol(SomeNumber, Decl(staticMemberAccessOffDerivedType1.ts, 5, 26))
+>P.GetNumber : Symbol(GetNumber, Decl(staticMemberAccessOffDerivedType1.ts, 0, 16))
 >P : Symbol(P, Decl(staticMemberAccessOffDerivedType1.ts, 4, 1))
->GetNumber : Symbol(SomeBase.GetNumber, Decl(staticMemberAccessOffDerivedType1.ts, 0, 16))
+>GetNumber : Symbol(GetNumber, Decl(staticMemberAccessOffDerivedType1.ts, 0, 16))
 }
 
