--- old.superCallArgsMustMatch.symbols
+++ new.superCallArgsMustMatch.symbols
@@ -6,11 +6,11 @@
 >T : Symbol(T, Decl(superCallArgsMustMatch.ts, 0, 9))
 
     public foo: T;
->foo : Symbol(T5.foo, Decl(superCallArgsMustMatch.ts, 0, 12))
+>foo : Symbol(foo, Decl(superCallArgsMustMatch.ts, 0, 12))
 >T : Symbol(T, Decl(superCallArgsMustMatch.ts, 0, 9))
 
     constructor(public bar: T) { }
->bar : Symbol(T5.bar, Decl(superCallArgsMustMatch.ts, 4, 16))
+>bar : Symbol(bar, Decl(superCallArgsMustMatch.ts, 4, 16))
 >T : Symbol(T, Decl(superCallArgsMustMatch.ts, 0, 9))
 
 }
@@ -30,9 +30,9 @@
 
         var x: number = this.foo;
 >x : Symbol(x, Decl(superCallArgsMustMatch.ts, 18, 11))
->this.foo : Symbol(T5.foo, Decl(superCallArgsMustMatch.ts, 0, 12))
+>this.foo : Symbol(foo, Decl(superCallArgsMustMatch.ts, 0, 12))
 >this : Symbol(T6, Decl(superCallArgsMustMatch.ts, 6, 1))
->foo : Symbol(T5.foo, Decl(superCallArgsMustMatch.ts, 0, 12))
+>foo : Symbol(foo, Decl(superCallArgsMustMatch.ts, 0, 12))
 
     }
 
