--- old.super1.symbols
+++ new.super1.symbols
@@ -6,7 +6,7 @@
 >Base1 : Symbol(Base1, Decl(super1.ts, 0, 0))
 
     public foo() {
->foo : Symbol(Base1.foo, Decl(super1.ts, 1, 13))
+>foo : Symbol(foo, Decl(super1.ts, 1, 13))
 
         return "base";
     }
@@ -17,7 +17,7 @@
 >Base1 : Symbol(Base1, Decl(super1.ts, 0, 0))
 
     public bar() {
->bar : Symbol(Sub1.bar, Decl(super1.ts, 7, 26))
+>bar : Symbol(bar, Decl(super1.ts, 7, 26))
 
         return "base";
     }
@@ -28,7 +28,7 @@
 >Sub1 : Symbol(Sub1, Decl(super1.ts, 5, 1))
 
     public bar() {
->bar : Symbol(SubSub1.bar, Decl(super1.ts, 13, 28))
+>bar : Symbol(bar, Decl(super1.ts, 13, 28))
 
         return super.super.foo;
 >super : Symbol(Sub1, Decl(super1.ts, 5, 1))
@@ -40,7 +40,7 @@
 >Base2 : Symbol(Base2, Decl(super1.ts, 17, 1))
 
     public foo() {
->foo : Symbol(Base2.foo, Decl(super1.ts, 20, 13))
+>foo : Symbol(foo, Decl(super1.ts, 20, 13))
 
         return "base";
     }
@@ -51,7 +51,7 @@
 >Base2 : Symbol(Base2, Decl(super1.ts, 17, 1))
 
     public bar() {
->bar : Symbol(SubE2.bar, Decl(super1.ts, 26, 27))
+>bar : Symbol(bar, Decl(super1.ts, 26, 27))
 
         return super.prototype.foo = null;
 >super : Symbol(Base2, Decl(super1.ts, 17, 1))
@@ -63,7 +63,7 @@
 >Base3 : Symbol(Base3, Decl(super1.ts, 30, 1))
 
     public foo() {
->foo : Symbol(Base3.foo, Decl(super1.ts, 33, 13))
+>foo : Symbol(foo, Decl(super1.ts, 33, 13))
 
         return "base";
     }
@@ -74,7 +74,7 @@
 >Base3 : Symbol(Base3, Decl(super1.ts, 30, 1))
 
     public bar() {
->bar : Symbol(SubE3.bar, Decl(super1.ts, 39, 27))
+>bar : Symbol(bar, Decl(super1.ts, 39, 27))
 
         return super.bar();
 >super : Symbol(Base3, Decl(super1.ts, 30, 1))
@@ -89,7 +89,7 @@
 >Sub4 : Symbol(Sub4, Decl(super1.ts, 46, 14))
 
         public x(){
->x : Symbol(Sub4.x, Decl(super1.ts, 47, 16))
+>x : Symbol(x, Decl(super1.ts, 47, 16))
 
             return "hello";
         }
@@ -100,12 +100,12 @@
 >Sub4 : Symbol(Sub4, Decl(super1.ts, 46, 14))
 
         public x(){
->x : Symbol(SubSub4.x, Decl(super1.ts, 53, 38))
+>x : Symbol(x, Decl(super1.ts, 53, 38))
 
             return super.x();
->super.x : Symbol(Sub4.x, Decl(super1.ts, 47, 16))
+>super.x : Symbol(x, Decl(super1.ts, 47, 16))
 >super : Symbol(Sub4, Decl(super1.ts, 46, 14))
->x : Symbol(Sub4.x, Decl(super1.ts, 47, 16))
+>x : Symbol(x, Decl(super1.ts, 47, 16))
         }
     }
     
@@ -113,7 +113,7 @@
 >Sub4E : Symbol(Sub4E, Decl(super1.ts, 57, 5))
 
         public x() {
->x : Symbol(Sub4E.x, Decl(super1.ts, 59, 24))
+>x : Symbol(x, Decl(super1.ts, 59, 24))
 
             return super.x();
         }
