--- old.superNoModifiersCrash.symbols
+++ new.superNoModifiersCrash.symbols
@@ -5,15 +5,13 @@
 >Parent : Symbol(Parent, Decl(File.js, 0, 0))
 
     initialize() {
->initialize : Symbol(Parent.initialize, Decl(File.js, 0, 14))
+>initialize : Symbol(initialize, Decl(File.js, 0, 14))
 
         super.initialize(...arguments)
 >arguments : Symbol(arguments)
 
         return this.asdf = ''
->this.asdf : Symbol(Parent.asdf, Decl(File.js, 3, 14))
 >this : Symbol(Parent, Decl(File.js, 0, 0))
->asdf : Symbol(Parent.asdf, Decl(File.js, 3, 14))
     }
   }
 
@@ -22,6 +20,6 @@
 >Parent : Symbol(Parent, Decl(File.js, 0, 0))
 
     initialize() {
->initialize : Symbol(Child.initialize, Decl(File.js, 7, 28))
+>initialize : Symbol(initialize, Decl(File.js, 7, 28))
     }
 }
