--- old.uniqueSymbolAllowsIndexInObjectWithIndexSignature.symbols
+++ new.uniqueSymbolAllowsIndexInObjectWithIndexSignature.symbols
@@ -10,7 +10,7 @@
 >I : Symbol(I, Decl(uniqueSymbolAllowsIndexInObjectWithIndexSignature.ts, 1, 45))
 
   [SYM]: 'sym';
->[SYM] : Symbol(I[SYM], Decl(uniqueSymbolAllowsIndexInObjectWithIndexSignature.ts, 3, 20))
+>[SYM] : Symbol([SYM], Decl(uniqueSymbolAllowsIndexInObjectWithIndexSignature.ts, 3, 20))
 >SYM : Symbol(SYM, Decl(uniqueSymbolAllowsIndexInObjectWithIndexSignature.ts, 1, 12))
 
   [x: string]: 'str';
