--- old.unusedImports2.symbols
+++ new.unusedImports2.symbols
@@ -5,7 +5,7 @@
 >Calculator : Symbol(Calculator, Decl(file1.ts, 0, 0))
 
     handleChar() {}
->handleChar : Symbol(Calculator.handleChar, Decl(file1.ts, 0, 25))
+>handleChar : Symbol(handleChar, Decl(file1.ts, 0, 25))
 }
 
 export function test() {
@@ -25,7 +25,7 @@
 >Calculator : Symbol(Calculator, Decl(file2.ts, 0, 8))
 
 x.handleChar();
->x.handleChar : Symbol(Calculator.handleChar, Decl(file1.ts, 0, 25))
+>x.handleChar : Symbol(handleChar, Decl(file1.ts, 0, 25))
 >x : Symbol(x, Decl(file2.ts, 3, 3))
->handleChar : Symbol(Calculator.handleChar, Decl(file1.ts, 0, 25))
+>handleChar : Symbol(handleChar, Decl(file1.ts, 0, 25))
 
