--- old.anyAssignableToEveryType.symbols
+++ new.anyAssignableToEveryType.symbols
@@ -8,7 +8,7 @@
 >C : Symbol(C, Decl(anyAssignableToEveryType.ts, 0, 11))
 
     foo: string;
->foo : Symbol(C.foo, Decl(anyAssignableToEveryType.ts, 2, 9))
+>foo : Symbol(foo, Decl(anyAssignableToEveryType.ts, 2, 9))
 }
 var ac: C;
 >ac : Symbol(ac, Decl(anyAssignableToEveryType.ts, 5, 3))
@@ -18,7 +18,7 @@
 >I : Symbol(I, Decl(anyAssignableToEveryType.ts, 5, 10))
 
     foo: string;
->foo : Symbol(I.foo, Decl(anyAssignableToEveryType.ts, 6, 13))
+>foo : Symbol(foo, Decl(anyAssignableToEveryType.ts, 6, 13))
 }
 var ai: I;
 >ai : Symbol(ai, Decl(anyAssignableToEveryType.ts, 9, 3))
@@ -26,7 +26,7 @@
 
 enum E { A }
 >E : Symbol(E, Decl(anyAssignableToEveryType.ts, 9, 10))
->A : Symbol(E.A, Decl(anyAssignableToEveryType.ts, 11, 8))
+>A : Symbol(A, Decl(anyAssignableToEveryType.ts, 11, 8))
 
 var ae: E;
 >ae : Symbol(ae, Decl(anyAssignableToEveryType.ts, 12, 3))
