--- old.awaitClassExpression_es5.symbols
+++ new.awaitClassExpression_es5.symbols
@@ -6,12 +6,12 @@
 
 declare var p: Promise<typeof C>;
 >p : Symbol(p, Decl(awaitClassExpression_es5.ts, 1, 11))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 >C : Symbol(C, Decl(awaitClassExpression_es5.ts, 0, 0))
 
 async function func(): Promise<void> {
 >func : Symbol(func, Decl(awaitClassExpression_es5.ts, 1, 33))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     class D extends (await p) {
 >D : Symbol(D, Decl(awaitClassExpression_es5.ts, 3, 38))
