--- old.awaitBinaryExpression5_es5.symbols
+++ new.awaitBinaryExpression5_es5.symbols
@@ -6,7 +6,7 @@
 
 declare var p: Promise<boolean>;
 >p : Symbol(p, Decl(awaitBinaryExpression5_es5.ts, 1, 11))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
 declare function before(): void;
 >before : Symbol(before, Decl(awaitBinaryExpression5_es5.ts, 1, 32))
@@ -16,7 +16,7 @@
 
 async function func(): Promise<void> {
 >func : Symbol(func, Decl(awaitBinaryExpression5_es5.ts, 3, 31))
->Promise : Symbol(Promise, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.promise.d.ts, --, --))
+>Promise : Symbol(Promise)
 
     before();
 >before : Symbol(before, Decl(awaitBinaryExpression5_es5.ts, 1, 32))
