--- old.classAbstractCrashedOnce.symbols
+++ new.classAbstractCrashedOnce.symbols
@@ -5,7 +5,7 @@
 >foo : Symbol(foo, Decl(classAbstractCrashedOnce.ts, 0, 0))
 
     protected abstract test();
->test : Symbol(foo.test, Decl(classAbstractCrashedOnce.ts, 0, 20))
+>test : Symbol(test, Decl(classAbstractCrashedOnce.ts, 0, 20))
 }
 
 class bar extends foo {
@@ -13,7 +13,7 @@
 >foo : Symbol(foo, Decl(classAbstractCrashedOnce.ts, 0, 0))
 
     test() {
->test : Symbol(bar.test, Decl(classAbstractCrashedOnce.ts, 4, 23))
+>test : Symbol(test, Decl(classAbstractCrashedOnce.ts, 4, 23))
 
         this.
 >this : Symbol(bar, Decl(classAbstractCrashedOnce.ts, 2, 1))
