--- old.constLocalsInFunctionExpressions.symbols
+++ new.constLocalsInFunctionExpressions.symbols
@@ -16,9 +16,9 @@
 
         const f = () => x.length;
 >f : Symbol(f, Decl(constLocalsInFunctionExpressions.ts, 5, 13))
->x.length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>x.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(constLocalsInFunctionExpressions.ts, 3, 9))
->length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
     }
 }
 
@@ -36,9 +36,9 @@
     }
     const f = () => x.length;
 >f : Symbol(f, Decl(constLocalsInFunctionExpressions.ts, 14, 9))
->x.length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>x.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(constLocalsInFunctionExpressions.ts, 10, 9))
->length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 }
 
 function f3() {
@@ -53,9 +53,9 @@
 
         const f = function() { return x.length; };
 >f : Symbol(f, Decl(constLocalsInFunctionExpressions.ts, 20, 13))
->x.length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>x.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(constLocalsInFunctionExpressions.ts, 18, 9))
->length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
     }
 }
 
@@ -73,9 +73,9 @@
     }
     const f = function() { return x.length; };
 >f : Symbol(f, Decl(constLocalsInFunctionExpressions.ts, 29, 9))
->x.length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>x.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(constLocalsInFunctionExpressions.ts, 25, 9))
->length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 }
 
 function f5() {
@@ -90,8 +90,8 @@
 
         const f = () => () => x.length;
 >f : Symbol(f, Decl(constLocalsInFunctionExpressions.ts, 35, 13))
->x.length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>x.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(constLocalsInFunctionExpressions.ts, 33, 9))
->length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
     }
 }
