--- old.decoratorMetadataWithTypeOnlyImport.symbols
+++ new.decoratorMetadataWithTypeOnlyImport.symbols
@@ -18,7 +18,7 @@
 >MyComponent : Symbol(MyComponent, Decl(component.ts, 2, 27))
 
     constructor(public Service: Service) {
->Service : Symbol(MyComponent.Service, Decl(component.ts, 6, 16))
+>Service : Symbol(Service, Decl(component.ts, 6, 16))
 >Service : Symbol(Service, Decl(component.ts, 0, 13))
     }
 
@@ -26,7 +26,7 @@
 >decorator : Symbol(decorator, Decl(component.ts, 2, 11))
 
     method(x: this) {
->method : Symbol(MyComponent.method, Decl(component.ts, 7, 5))
+>method : Symbol(method, Decl(component.ts, 7, 5))
 >x : Symbol(x, Decl(component.ts, 10, 11))
     }
 }
