--- old.exportedEnumTypeAndValue.symbols
+++ new.exportedEnumTypeAndValue.symbols
@@ -3,7 +3,7 @@
 === def.js ===
 /** @enum {number} */
 const MyEnum = {
->MyEnum : Symbol(MyEnum, Decl(def.js, 1, 5), Decl(def.js, 0, 4))
+>MyEnum : Symbol(MyEnum, Decl(def.js, 1, 5))
 
   a: 1,
 >a : Symbol(a, Decl(def.js, 1, 16))
@@ -13,7 +13,7 @@
 
 };
 export default MyEnum;
->MyEnum : Symbol(MyEnum, Decl(def.js, 1, 5), Decl(def.js, 0, 4))
+>MyEnum : Symbol(MyEnum, Decl(def.js, 1, 5))
 
 === use.js ===
 import MyEnum from "./def";
