--- old.importCallExpressionShouldNotGetParen.symbols
+++ new.importCallExpressionShouldNotGetParen.symbols
@@ -5,9 +5,7 @@
 >localeName : Symbol(localeName, Decl(importCallExpressionShouldNotGetParen.ts, 0, 5))
 
 import(`./locales/${localeName}.js`).then(bar => {
->import(`./locales/${localeName}.js`).then : Symbol(Promise.then, Decl(lib.es5.d.ts, --, --))
 >localeName : Symbol(localeName, Decl(importCallExpressionShouldNotGetParen.ts, 0, 5))
->then : Symbol(Promise.then, Decl(lib.es5.d.ts, --, --))
 >bar : Symbol(bar, Decl(importCallExpressionShouldNotGetParen.ts, 1, 42))
 
     let x = bar;
@@ -17,9 +15,7 @@
 });
 
 import("./locales/" + localeName + ".js").then(bar => {
->import("./locales/" + localeName + ".js").then : Symbol(Promise.then, Decl(lib.es5.d.ts, --, --))
 >localeName : Symbol(localeName, Decl(importCallExpressionShouldNotGetParen.ts, 0, 5))
->then : Symbol(Promise.then, Decl(lib.es5.d.ts, --, --))
 >bar : Symbol(bar, Decl(importCallExpressionShouldNotGetParen.ts, 5, 47))
 
     let x = bar;
