--- old.inlineJsxAndJsxFragPragma.symbols
+++ new.inlineJsxAndJsxFragPragma.symbols
@@ -2,7 +2,7 @@
 
 === renderer.d.ts ===
 declare global {
->global : Symbol(global, Decl(renderer.d.ts, 0, 0))
+>global : Symbol((Missing), Decl(renderer.d.ts, 0, 0))
 
     namespace JSX {
 >JSX : Symbol(JSX, Decl(renderer.d.ts, 0, 16))
@@ -34,8 +34,6 @@
 >Fragment : Symbol(Fragment, Decl(preacty.tsx, 4, 10))
 
 <><div></div></>
->div : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
 
 === snabbdomy.tsx ===
 /* @jsx jsx */
@@ -44,8 +42,6 @@
 >jsx : Symbol(jsx, Decl(snabbdomy.tsx, 2, 8))
 
 <><span></span></>
->span : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
->span : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
 
 === preacty-only-fragment.tsx ===
 /**
@@ -93,8 +89,6 @@
 >Fragment : Symbol(Fragment, Decl(preacty-no-fragment.tsx, 4, 10))
 
 <div></div>
->div : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
 
 === snabbdomy-no-fragment.tsx ===
 /* @jsx jsx */
@@ -103,8 +97,6 @@
 >jsx : Symbol(jsx, Decl(snabbdomy-no-fragment.tsx, 2, 8))
 
 <div></div>
->div : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(renderer.d.ts, 2, 37))
 
 === preacty-only-component.tsx ===
 /**
