--- old.inlineJsxAndJsxFragPragmaOverridesCompilerOptions.symbols
+++ new.inlineJsxAndJsxFragPragmaOverridesCompilerOptions.symbols
@@ -2,7 +2,7 @@
 
 === react.d.ts ===
 declare global {
->global : Symbol(global, Decl(react.d.ts, 0, 0))
+>global : Symbol((Missing), Decl(react.d.ts, 0, 0))
 
     namespace JSX {
 >JSX : Symbol(JSX, Decl(react.d.ts, 0, 16))
@@ -38,8 +38,6 @@
 >Fragment : Symbol(Fragment, Decl(reacty.tsx, 0, 22))
 
 <><span></span></>
->span : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
->span : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
 
 === preacty.tsx ===
 /**
@@ -51,8 +49,6 @@
 >Frag : Symbol(Frag, Decl(preacty.tsx, 4, 10))
 
 <><div></div></>
->div : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
 
 === snabbdomy.tsx ===
 /**
@@ -63,8 +59,6 @@
 >h : Symbol(h, Decl(snabbdomy.tsx, 4, 8))
 
 <><div></div></>
->div : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
 
 === mix-n-match.tsx ===
 /* @jsx h */
@@ -76,6 +70,3 @@
 >Fragment : Symbol(Fragment, Decl(mix-n-match.tsx, 3, 8))
 
 <><span></span></>
->span : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
->span : Symbol(JSX.IntrinsicElements.__index, Decl(react.d.ts, 2, 37))
-
