--- old.jsDeclarationsUniqueSymbolUsage.symbols
+++ new.jsDeclarationsUniqueSymbolUsage.symbols
@@ -3,7 +3,6 @@
 === a.js ===
 export const kSymbol = Symbol("my-symbol");
 >kSymbol : Symbol(kSymbol, Decl(a.js, 0, 12))
->Symbol : Symbol(Symbol, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.symbol.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --))
 
 /**
  * @typedef {{[kSymbol]: true}} WithSymbol
