--- old.jsdocImplements_properties.symbols
+++ new.jsdocImplements_properties.symbols
@@ -3,9 +3,7 @@
 === /a.js ===
 class A { constructor() { this.x = 0; } }
 >A : Symbol(A, Decl(a.js, 0, 0))
->this.x : Symbol(A.x, Decl(a.js, 0, 25))
 >this : Symbol(A, Decl(a.js, 0, 0))
->x : Symbol(A.x, Decl(a.js, 0, 25))
 
 /** @implements A*/
 class B {}
@@ -16,7 +14,7 @@
 >B2 : Symbol(B2, Decl(a.js, 2, 10))
 
     x = 10
->x : Symbol(B2.x, Decl(a.js, 5, 10))
+>x : Symbol(x, Decl(a.js, 5, 10))
 }
 
 /** @implements {A}*/
@@ -24,8 +22,6 @@
 >B3 : Symbol(B3, Decl(a.js, 7, 1))
 
     constructor() { this.x = 10 }
->this.x : Symbol(B3.x, Decl(a.js, 11, 19))
 >this : Symbol(B3, Decl(a.js, 7, 1))
->x : Symbol(B3.x, Decl(a.js, 11, 19))
 }
 
