--- old.lateBoundClassMemberAssignmentJS2.symbols
+++ new.lateBoundClassMemberAssignmentJS2.symbols
@@ -14,7 +14,7 @@
     }
 
     method() {
->method : Symbol(MyClass.method, Decl(lateBoundClassMemberAssignmentJS2.js, 4, 5))
+>method : Symbol(method, Decl(lateBoundClassMemberAssignmentJS2.js, 4, 5))
 
         this[_sym] = "yep";
 >this : Symbol(MyClass, Decl(lateBoundClassMemberAssignmentJS2.js, 0, 27))
