--- old.localTypes5.symbols
+++ new.localTypes5.symbols
@@ -9,7 +9,7 @@
 >X : Symbol(X, Decl(localTypes5.ts, 0, 19))
 
         m<B, C>() {
->m : Symbol(X.m, Decl(localTypes5.ts, 1, 13))
+>m : Symbol(m, Decl(localTypes5.ts, 1, 13))
 >B : Symbol(B, Decl(localTypes5.ts, 2, 10))
 >C : Symbol(C, Decl(localTypes5.ts, 2, 12))
 
@@ -32,9 +32,9 @@
 >X : Symbol(X, Decl(localTypes5.ts, 0, 19))
 
     return x.m<number, boolean>();
->x.m : Symbol(X.m, Decl(localTypes5.ts, 1, 13))
+>x.m : Symbol(m, Decl(localTypes5.ts, 1, 13))
 >x : Symbol(x, Decl(localTypes5.ts, 10, 7))
->m : Symbol(X.m, Decl(localTypes5.ts, 1, 13))
+>m : Symbol(m, Decl(localTypes5.ts, 1, 13))
 }
 var x = foo<void>();
 >x : Symbol(x, Decl(localTypes5.ts, 13, 3))
