--- old.nodeModules1(module=node16).symbols
+++ new.nodeModules1(module=node16).symbols
@@ -318,7 +318,7 @@
 >x : Symbol(x, Decl(index.mts, 86, 5))
 
 export {x};
->x : Symbol(m2.x, Decl(index.mts, 87, 8))
+>x : Symbol(x, Decl(index.mts, 87, 8))
 
 === index.cts ===
 // ESM-format imports below should issue errors
@@ -567,7 +567,7 @@
 >x : Symbol(x, Decl(index.cts, 86, 5))
 
 export {x};
->x : Symbol(m3.x, Decl(index.cts, 87, 8))
+>x : Symbol(x, Decl(index.cts, 87, 8))
 
 === index.ts ===
 import * as m1 from "./index.js";
@@ -815,5 +815,5 @@
 >x : Symbol(x, Decl(index.ts, 85, 5))
 
 export {x};
->x : Symbol(m1.x, Decl(index.ts, 86, 8))
+>x : Symbol(x, Decl(index.ts, 86, 8))
 
