--- old.objectTypesWithOptionalProperties.symbols
+++ new.objectTypesWithOptionalProperties.symbols
@@ -14,14 +14,14 @@
 >I : Symbol(I, Decl(objectTypesWithOptionalProperties.ts, 4, 1))
 
     x?: number; // ok
->x : Symbol(I.x, Decl(objectTypesWithOptionalProperties.ts, 6, 13))
+>x : Symbol(x, Decl(objectTypesWithOptionalProperties.ts, 6, 13))
 }
 
 class C {
 >C : Symbol(C, Decl(objectTypesWithOptionalProperties.ts, 8, 1))
 
     x?: number; // ok
->x : Symbol(C.x, Decl(objectTypesWithOptionalProperties.ts, 10, 9))
+>x : Symbol(x, Decl(objectTypesWithOptionalProperties.ts, 10, 9))
 }
 
 interface I2<T> {
@@ -29,7 +29,7 @@
 >T : Symbol(T, Decl(objectTypesWithOptionalProperties.ts, 14, 13))
 
     x?: T; // ok
->x : Symbol(I2.x, Decl(objectTypesWithOptionalProperties.ts, 14, 17))
+>x : Symbol(x, Decl(objectTypesWithOptionalProperties.ts, 14, 17))
 >T : Symbol(T, Decl(objectTypesWithOptionalProperties.ts, 14, 13))
 }
 
@@ -38,7 +38,7 @@
 >T : Symbol(T, Decl(objectTypesWithOptionalProperties.ts, 18, 9))
 
     x?: T; // ok
->x : Symbol(C2.x, Decl(objectTypesWithOptionalProperties.ts, 18, 13))
+>x : Symbol(x, Decl(objectTypesWithOptionalProperties.ts, 18, 13))
 >T : Symbol(T, Decl(objectTypesWithOptionalProperties.ts, 18, 9))
 }
 
