--- old.parserComputedPropertyName29.symbols
+++ new.parserComputedPropertyName29.symbols
@@ -6,8 +6,8 @@
 
     // yes ASI
     [e] = id++
->[e] : Symbol(C[e], Decl(parserComputedPropertyName29.ts, 0, 9))
+>[e] : Symbol([e], Decl(parserComputedPropertyName29.ts, 0, 9))
 
     [e2]: number
->[e2] : Symbol(C[e2], Decl(parserComputedPropertyName29.ts, 2, 14))
+>[e2] : Symbol([e2], Decl(parserComputedPropertyName29.ts, 2, 14))
 }
