--- old.privateNameLateSuperUseDefineForClassFields(target=es2022).symbols
+++ new.privateNameLateSuperUseDefineForClassFields(target=es2022).symbols
@@ -9,7 +9,7 @@
 >B : Symbol(B, Decl(privateNameLateSuperUseDefineForClassFields.ts, 0, 0))
 
     #x;
->#x : Symbol(A.#x, Decl(privateNameLateSuperUseDefineForClassFields.ts, 1, 19))
+>#x : Symbol(#x, Decl(privateNameLateSuperUseDefineForClassFields.ts, 1, 19))
 
     constructor() {
         void 0;
