--- old.privateNameBadSuperUseDefineForClassFields(target=es2022).symbols
+++ new.privateNameBadSuperUseDefineForClassFields(target=es2022).symbols
@@ -9,7 +9,7 @@
 >B : Symbol(B, Decl(privateNameBadSuperUseDefineForClassFields.ts, 0, 0))
 
   #x;
->#x : Symbol(A.#x, Decl(privateNameBadSuperUseDefineForClassFields.ts, 1, 19))
+>#x : Symbol(#x, Decl(privateNameBadSuperUseDefineForClassFields.ts, 1, 19))
 
   constructor() {
     this;
