--- old.privateStaticNotAccessibleInClodule.symbols
+++ new.privateStaticNotAccessibleInClodule.symbols
@@ -7,10 +7,10 @@
 >C : Symbol(C, Decl(privateStaticNotAccessibleInClodule.ts, 0, 0), Decl(privateStaticNotAccessibleInClodule.ts, 5, 1))
 
     private foo: string;
->foo : Symbol(C.foo, Decl(privateStaticNotAccessibleInClodule.ts, 2, 9))
+>foo : Symbol(foo, Decl(privateStaticNotAccessibleInClodule.ts, 2, 9))
 
     private static bar: string;
->bar : Symbol(C.bar, Decl(privateStaticNotAccessibleInClodule.ts, 3, 24))
+>bar : Symbol(bar, Decl(privateStaticNotAccessibleInClodule.ts, 3, 24))
 }
 
 module C {
@@ -18,7 +18,7 @@
 
     export var y = C.bar; // error
 >y : Symbol(y, Decl(privateStaticNotAccessibleInClodule.ts, 8, 14))
->C.bar : Symbol(C.bar, Decl(privateStaticNotAccessibleInClodule.ts, 3, 24))
+>C.bar : Symbol(bar, Decl(privateStaticNotAccessibleInClodule.ts, 3, 24))
 >C : Symbol(C, Decl(privateStaticNotAccessibleInClodule.ts, 0, 0), Decl(privateStaticNotAccessibleInClodule.ts, 5, 1))
->bar : Symbol(C.bar, Decl(privateStaticNotAccessibleInClodule.ts, 3, 24))
+>bar : Symbol(bar, Decl(privateStaticNotAccessibleInClodule.ts, 3, 24))
 }
