--- old.subtypingWithObjectMembersOptionality2.symbols
+++ new.subtypingWithObjectMembersOptionality2.symbols
@@ -5,18 +5,18 @@
 
 interface Base { foo: string; }
 >Base : Symbol(Base, Decl(subtypingWithObjectMembersOptionality2.ts, 0, 0))
->foo : Symbol(Base.foo, Decl(subtypingWithObjectMembersOptionality2.ts, 2, 16))
+>foo : Symbol(foo, Decl(subtypingWithObjectMembersOptionality2.ts, 2, 16))
 
 interface Derived extends Base { bar: string; }
 >Derived : Symbol(Derived, Decl(subtypingWithObjectMembersOptionality2.ts, 2, 31))
 >Base : Symbol(Base, Decl(subtypingWithObjectMembersOptionality2.ts, 0, 0))
->bar : Symbol(Derived.bar, Decl(subtypingWithObjectMembersOptionality2.ts, 3, 32))
+>bar : Symbol(bar, Decl(subtypingWithObjectMembersOptionality2.ts, 3, 32))
 
 interface T {
 >T : Symbol(T, Decl(subtypingWithObjectMembersOptionality2.ts, 3, 47))
 
     Foo: Base;
->Foo : Symbol(T.Foo, Decl(subtypingWithObjectMembersOptionality2.ts, 5, 13))
+>Foo : Symbol(Foo, Decl(subtypingWithObjectMembersOptionality2.ts, 5, 13))
 >Base : Symbol(Base, Decl(subtypingWithObjectMembersOptionality2.ts, 0, 0))
 }
 
@@ -25,7 +25,7 @@
 >T : Symbol(T, Decl(subtypingWithObjectMembersOptionality2.ts, 3, 47))
 
     Foo?: Derived // error
->Foo : Symbol(S.Foo, Decl(subtypingWithObjectMembersOptionality2.ts, 9, 23))
+>Foo : Symbol(Foo, Decl(subtypingWithObjectMembersOptionality2.ts, 9, 23))
 >Derived : Symbol(Derived, Decl(subtypingWithObjectMembersOptionality2.ts, 2, 31))
 }
 
@@ -33,7 +33,7 @@
 >T2 : Symbol(T2, Decl(subtypingWithObjectMembersOptionality2.ts, 11, 1))
 
     1: Base;
->1 : Symbol(T2[1], Decl(subtypingWithObjectMembersOptionality2.ts, 13, 14))
+>1 : Symbol(1, Decl(subtypingWithObjectMembersOptionality2.ts, 13, 14))
 >Base : Symbol(Base, Decl(subtypingWithObjectMembersOptionality2.ts, 0, 0))
 }
 
@@ -42,7 +42,7 @@
 >T2 : Symbol(T2, Decl(subtypingWithObjectMembersOptionality2.ts, 11, 1))
 
     1?: Derived; // error
->1 : Symbol(S2[1], Decl(subtypingWithObjectMembersOptionality2.ts, 17, 25))
+>1 : Symbol(1, Decl(subtypingWithObjectMembersOptionality2.ts, 17, 25))
 >Derived : Symbol(Derived, Decl(subtypingWithObjectMembersOptionality2.ts, 2, 31))
 }
 
@@ -50,7 +50,7 @@
 >T3 : Symbol(T3, Decl(subtypingWithObjectMembersOptionality2.ts, 19, 1))
 
     '1': Base;
->'1' : Symbol(T3['1'], Decl(subtypingWithObjectMembersOptionality2.ts, 21, 14))
+>'1' : Symbol('1', Decl(subtypingWithObjectMembersOptionality2.ts, 21, 14))
 >Base : Symbol(Base, Decl(subtypingWithObjectMembersOptionality2.ts, 0, 0))
 }
 
@@ -59,7 +59,7 @@
 >T3 : Symbol(T3, Decl(subtypingWithObjectMembersOptionality2.ts, 19, 1))
 
     '1'?: Derived; // error
->'1' : Symbol(S3['1'], Decl(subtypingWithObjectMembersOptionality2.ts, 25, 25))
+>'1' : Symbol('1', Decl(subtypingWithObjectMembersOptionality2.ts, 25, 25))
 >Derived : Symbol(Derived, Decl(subtypingWithObjectMembersOptionality2.ts, 2, 31))
 }
 
