--- old.symbolProperty52.symbols
+++ new.symbolProperty52.symbols
@@ -6,7 +6,6 @@
 
     [Symbol.nonsense]: 0
 >[Symbol.nonsense] : Symbol([Symbol.nonsense], Decl(symbolProperty52.ts, 0, 11))
->Symbol : Symbol(Symbol, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.symbol.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --))
 
 };
 
@@ -15,5 +14,4 @@
 
 obj[Symbol.nonsense];
 >obj : Symbol(obj, Decl(symbolProperty52.ts, 0, 3))
->Symbol : Symbol(Symbol, Decl(lib.es5.d.ts, --, --), Decl(lib.es2015.symbol.d.ts, --, --), Decl(lib.es2015.symbol.wellknown.d.ts, --, --))
 
