--- old.tsNoCheckForTypescriptComments2.symbols
+++ new.tsNoCheckForTypescriptComments2.symbols
@@ -10,7 +10,7 @@
 >Aleph : Symbol(Aleph, Decl(file.ts, 2, 24))
 
     q: number;
->q : Symbol(Aleph.q, Decl(file.ts, 4, 24))
+>q : Symbol(q, Decl(file.ts, 4, 24))
 }
 
 export class Bet implements Aleph {
@@ -18,6 +18,6 @@
 >Aleph : Symbol(Aleph, Decl(file.ts, 2, 24))
 
     q: string = "lol" // And so will this implements error
->q : Symbol(Bet.q, Decl(file.ts, 8, 35))
+>q : Symbol(q, Decl(file.ts, 8, 35))
 }
 
