--- old.tsxAttributeResolution14.symbols
+++ new.tsxAttributeResolution14.symbols
@@ -11,11 +11,11 @@
 >IntrinsicElements : Symbol(IntrinsicElements, Decl(react.d.ts, 1, 22))
 
         div: any;
->div : Symbol(IntrinsicElements.div, Decl(react.d.ts, 2, 30))
+>div : Symbol(div, Decl(react.d.ts, 2, 30))
 	}
     interface ElementAttributesProperty { prop: any }
 >ElementAttributesProperty : Symbol(ElementAttributesProperty, Decl(react.d.ts, 4, 2))
->prop : Symbol(ElementAttributesProperty.prop, Decl(react.d.ts, 5, 41))
+>prop : Symbol(prop, Decl(react.d.ts, 5, 41))
 }
 
 === file.tsx ===
@@ -23,7 +23,7 @@
 >IProps : Symbol(IProps, Decl(file.tsx, 0, 0))
 
   primaryText: string,
->primaryText : Symbol(IProps.primaryText, Decl(file.tsx, 0, 18))
+>primaryText : Symbol(primaryText, Decl(file.tsx, 0, 18))
 
   [propName: string]: string | number
 >propName : Symbol(propName, Decl(file.tsx, 2, 3))
@@ -35,8 +35,6 @@
 >IProps : Symbol(IProps, Decl(file.tsx, 0, 0))
 
   return <div>props.primaryText</div>
->div : Symbol(JSX.IntrinsicElements.div, Decl(react.d.ts, 2, 30))
->div : Symbol(JSX.IntrinsicElements.div, Decl(react.d.ts, 2, 30))
 }
 
 function VerticalNav() {
@@ -44,8 +42,6 @@
 
   return (
     <div>
->div : Symbol(JSX.IntrinsicElements.div, Decl(react.d.ts, 2, 30))
-
       <VerticalNavMenuItem primaryText={2} />  // error
 >VerticalNavMenuItem : Symbol(VerticalNavMenuItem, Decl(file.tsx, 3, 1))
 >primaryText : Symbol(primaryText, Decl(file.tsx, 12, 26))
@@ -61,7 +57,5 @@
 >primaryText : Symbol(primaryText, Decl(file.tsx, 14, 49))
 
     </div>
->div : Symbol(JSX.IntrinsicElements.div, Decl(react.d.ts, 2, 30))
-
   )
 } 
