--- old.tsxAttributeResolution8.symbols
+++ new.tsxAttributeResolution8.symbols
@@ -11,7 +11,7 @@
 >IntrinsicElements : Symbol(IntrinsicElements, Decl(file.tsx, 1, 22))
 
 		test1: {x: string};
->test1 : Symbol(IntrinsicElements.test1, Decl(file.tsx, 2, 30))
+>test1 : Symbol(test1, Decl(file.tsx, 2, 30))
 >x : Symbol(x, Decl(file.tsx, 3, 10))
 	}
 }
@@ -21,6 +21,5 @@
 
 // Should be OK
 <test1 {...x} />
->test1 : Symbol(JSX.IntrinsicElements.test1, Decl(file.tsx, 2, 30))
 >x : Symbol(x, Decl(file.tsx, 7, 3))
 
