--- old.tsxReactEmitWhitespace2.symbols
+++ new.tsxReactEmitWhitespace2.symbols
@@ -19,22 +19,9 @@
 
 // Emit ' word' in the last string
 <div>word <code>code</code> word</div>;
->div : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->code : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->code : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
-
 // Same here
 <div><code>code</code> word</div>;
->div : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->code : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->code : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
-
 // And here
 <div><code /> word</div>;
->div : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->code : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
->div : Symbol(JSX.IntrinsicElements.__index, Decl(file.tsx, 2, 30))
 
 
