--- old.typeGuardOfFormTypeOfFunction.symbols
+++ new.typeGuardOfFormTypeOfFunction.symbols
@@ -152,9 +152,9 @@
 >item : Symbol(item, Decl(typeGuardOfFormTypeOfFunction.ts, 67, 13))
 
             item.call(obj);
->item.call : Symbol(Function.call, Decl(lib.es5.d.ts, --, --))
+>item.call : Symbol(call, Decl(lib.es5.d.ts, --, --))
 >item : Symbol(item, Decl(typeGuardOfFormTypeOfFunction.ts, 67, 13))
->call : Symbol(Function.call, Decl(lib.es5.d.ts, --, --))
+>call : Symbol(call, Decl(lib.es5.d.ts, --, --))
 >obj : Symbol(obj, Decl(typeGuardOfFormTypeOfFunction.ts, 65, 36))
     }
 }
