--- old.typeGuardsTypeParameters.symbols
+++ new.typeGuardsTypeParameters.symbols
@@ -7,7 +7,7 @@
 >C : Symbol(C, Decl(typeGuardsTypeParameters.ts, 0, 0))
 
     prop: string;
->prop : Symbol(C.prop, Decl(typeGuardsTypeParameters.ts, 2, 9))
+>prop : Symbol(prop, Decl(typeGuardsTypeParameters.ts, 2, 9))
 }
 
 function f1<T>(x: T) {
@@ -31,9 +31,9 @@
 >x : Symbol(x, Decl(typeGuardsTypeParameters.ts, 6, 15))
 
         x.prop;
->x.prop : Symbol(C.prop, Decl(typeGuardsTypeParameters.ts, 2, 9))
+>x.prop : Symbol(prop, Decl(typeGuardsTypeParameters.ts, 2, 9))
 >x : Symbol(x, Decl(typeGuardsTypeParameters.ts, 6, 15))
->prop : Symbol(C.prop, Decl(typeGuardsTypeParameters.ts, 2, 9))
+>prop : Symbol(prop, Decl(typeGuardsTypeParameters.ts, 2, 9))
     }
 }
 
@@ -56,9 +56,9 @@
 >x : Symbol(x, Decl(typeGuardsTypeParameters.ts, 14, 15))
 
         x.length;
->x.length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>x.length : Symbol(length, Decl(lib.es5.d.ts, --, --))
 >x : Symbol(x, Decl(typeGuardsTypeParameters.ts, 14, 15))
->length : Symbol(String.length, Decl(lib.es5.d.ts, --, --))
+>length : Symbol(length, Decl(lib.es5.d.ts, --, --))
     }
 }
 
@@ -89,9 +89,9 @@
 >value : Symbol(value, Decl(typeGuardsTypeParameters.ts, 27, 13))
 
             strings.push(value);
->strings.push : Symbol(Array.push, Decl(lib.es5.d.ts, --, --))
+>strings.push : Symbol(push, Decl(lib.es5.d.ts, --, --))
 >strings : Symbol(strings, Decl(typeGuardsTypeParameters.ts, 25, 9))
->push : Symbol(Array.push, Decl(lib.es5.d.ts, --, --))
+>push : Symbol(push, Decl(lib.es5.d.ts, --, --))
 >value : Symbol(value, Decl(typeGuardsTypeParameters.ts, 27, 13))
         }
     }
